common/index.ts

```diff
@@ -38,6 +38,19 @@ export const OPENID_AUTH_LOGOUT = '/auth/openid/logout';
 export const SAML_AUTH_LOGOUT = '/auth/saml/logout';
 export const ANONYMOUS_AUTH_LOGOUT = '/auth/anonymous/logout';
 
+export const ANONYMOUS_ROUTES = [
+  LOGIN_PAGE_URI,
+  CUSTOM_ERROR_PAGE_URI,
+  API_AUTH_LOGIN,
+  API_AUTH_LOGOUT,
+  OPENID_AUTH_LOGIN,
+  ANONYMOUS_AUTH_LOGIN,
+  OPENID_AUTH_LOGOUT,
+  SAML_AUTH_LOGOUT,
+  ANONYMOUS_AUTH_LOGOUT,
+  SAML_AUTH_LOGIN,
```

Comment:
is this a verified exhaustive list of anonymous routes?

Reply:
@DarshitChanpura I'm on it verifying

Reply:
@DarshitChanpura FYI: I was hinted that maybe there is a possibility to check from the route itself, and this list would be a backup. I'm on it.

Reply:
This was resolved by `isAnonymousPage()` function in a more recent commit